AssertionError in test_git_dirty_notag in UTC-N after N before midnight

When I run tests on master, I get a failure in `test_git`:

```
FAILED testing/test_git.py::test_git_dirty_notag - AssertionError: assert '.d20200506' in '0.1.dev1+ge5adfd7.d20200507'
```

It's currently May 6 where I am but May 7 in UTC.

I suspect the issue is that git is generating dates in UTC but the tests are using a local date.

--- a/testing/test_git.py
+++ b/testing/test_git.py
@@ -4,7 +4,7 @@ from setuptools_scm import integration
 from setuptools_scm.utils import do
 from setuptools_scm import git
 import pytest
-from datetime import date
+from datetime import datetime
 from os.path import join as opj
 from setuptools_scm.file_finder_git import git_find_files
 
@@ -113,7 +113,8 @@ def test_git_dirty_notag(wd):
     wd.write("test.txt", "test2")
     wd("git add test.txt")
     assert wd.version.startswith("0.1.dev1")
-    today = date.today()
+    # the date on the tag is in UTC
+    today = datetime.utcnow().date()
     # we are dirty, check for the tag
     assert today.strftime(".d%Y%m%d") in wd.version
 

